todoman/ui.py

```diff
@@ -224,7 +219,14 @@ def compact_multiple(self, todos):
             percent = todo.percent_complete or ''
             if percent:
                 percent = " ({}%)".format(percent)
-            urgent = " " if todo.priority in [None, 0] else "!"
+            if todo.priority == 5:
+                priority = "!!"
+            elif todo.priority == 4:
```

Comment:
I thínk this sort of validation should happen somewhere else, so I think it's not necessary. I filed https://github.com/pimutils/todoman/issues/140 because we haven't thought about this.